test/CMakeLists.txt

```diff
@@ -1,5 +1,18 @@
 # SPDX-License-Identifier: Apache-2.0
 
+# Valid optimization levels are 0, 1, 2, and 3
+# Default to 3 if ONNX_MLIR_TEST_OPTLEVEL not set or cached
+set(OPTLEVELS 0 1 2 3)
```

Comment:
I think it would be better to have the level specified as OX rather than just X. Otherwise it's a little weird.

Reply:
It's done this way to be consistent with the way the optlevel is set for backend test.py via `-O {0,1,2,3}`.